src/lib/salad/rtree.h

```diff
@@ -116,6 +120,13 @@ enum rtree_distance_type {
 	RTREE_MANHATTAN = 1 /* Manhattan distance, fabs(dx) + fabs(dy) */
 };
 
+/* Neighbor pagination mode */
+enum position_type {
+	NO_PAGINATION = 0, /* No rtree pagination */
+	SET_POS = 1, /* new position set */
+	FIND_POS = 2 /* find pos tuple in rtree */
```

Comment:
Why do you need it?
AFAICS, `FIND_POS` is not needed at all - you only use it once, and that usage has no effect.

Reply:
Please, address this comment.